src/Microsoft.FileFormats/PDB/PDBStructures.cs

```diff
@@ -33,9 +33,9 @@ private static byte[] ExpectedMagic
         public uint Reserved;
 
         #region Validation Rules
-        public ValidationRule IsMagicValid
+        public bool IsMagicValid
```

Comment:
Even though no one outside of this project should be using this property, it is public so there could be.

Reply:
The better change would probably be to change all of the structs in `PDBStructure.cs` to `internal`.  Would you like me to do that?

Reply:
I looked into reducing visibility further, and it seems worse than what stands there now.  I think this is fine.